tests/dualtor/test_orchagent_standby_tor_downstream.py

```diff
@@ -225,7 +221,6 @@ def test_standby_tor_downstream_loopback_route_readded(
     """
     Verify traffic is equally distributed via loopback route
     """
-    pt_require('dualtor' in tbinfo['topo']['name'], "Only run on dualtor testbed")
```

Comment:
@xwjiang-ms I didn't see test_standby_tor_downstream_loopback_route_readded in condition yaml file. Should be skipped on other testbeds except real dualtor?

Reply:
I think we can run this case in real dualtor since the original code didn't skip it, and we can find success/fail test results in Kusto.